runelite-client/src/main/java/net/runelite/client/plugins/npchighlight/NpcIndicatorsConfig.java

```diff
@@ -107,4 +107,14 @@ default boolean showRespawnTimer()
 	{
 		return false;
 	}
+	
+	@ConfigItem(
+		position = 7,
+		keyName = "drawDistance",
+		name = "Monster Render Distance",
+		description = "The distance that monsters will be highlighted up to.")
+	default int drawDistance()
+	{
+		return 3;
```

Comment:
This should not default to 3. If merged, other people have to use your code too, and most people would consider this to be broken.